docs/introduction/concepts.md

```diff
@@ -31,3 +31,27 @@ Producing and exchange olive oil and coffee, even more simplified.
 ### ... to thinking bigger.
 
 ![river flow](https://rawgit.com/valueflows/valueflows/master/release-doc-in-process/vf-river.png)
+
+### Transversing value flows
+
+Value flows can be transversed forwards ("tracing") or backwards ("tracking").  Often people use the term "provenance" when looking backwards towards the source of some resource, sometimes when a problem emerges (as in a disease outbreak from food), sometimes to know the quality of the resource (as in wanting local humanely produced food with lower ecological impact).
+
+The data structures of ValueFlows provide the ability to trace or track any value flow, no matter how long or complex:
+
+* For an EconomicResource:
+    * after: EconomicEvents that are inputs to Processes or Transfers
+    * before: EconomicEvents that are outputs from Processes or Transfers
+* For a Process or Transfer:
+    * after: EconomicEvents that are outputs
+    * before: EconomicEvents that are inputs
+* For an EconomicEvent:
+    * after: 
+        * a Process or Transfer to which it is an input, or
+        * an EconomicResource which it incremented
```

Comment:
I think it may need to match *before*:
 if it is an output to a Process or Transfer, the EconomicResource which it affects.

Not all output actions on a resource have to increment it - eg. *improve*

Reply:
good point, will fix